### ISSUE TITLE
Show "Time left"-counter in the fullscreen modeling view

### ISSUE THREAD

**Issue by florian-glombik** [ls1intum/Artemis#1909]
### Is your feature request related to a problem?

During an exam, a student would always like to see how much time is left. Therefore the _Time left_ counter exists, however, this counter is not visible when entering the full-screen mode.
Since there is much space left in the full-screen mode, it would be good if the timer would also be visible there.

![Exam - Google Chrome 13 07 2020 16_05_51](https://user-images.githubusercontent.com/63976129/87792681-8866ef00-c844-11ea-8ed2-77cc64c53b01.png)

### Describe the solution you'd like

_Time left_ counter added at the upper right corner of the screen, to the left side of _Exit Fullscreen_.

### Describe alternatives you've considered

It is part of an exam to keep track of the time and organize yourself as a student; however, in a normal exam you would always see the clock and I think that the students should be able to concentrate on the task instead of having to leave the full-screen mode to see how much time is left.

**Comment by anditurdiu** [comment]
I just analysed this bug and the complaint feature seems to work as expected. 
I logged in with test user 7 to view the exam you mentioned and indeed under the complaints section, the submission is not listed.
![image](https://user-images.githubusercontent.com/15110960/87871695-db5eb480-c9b2-11ea-90b6-ec33c5d473d9.png)
It's unclear to me, why it shows up under student submissions though. I am looking into it
